### PR TITLE
Rn2244

### PR DESCRIPTION
**Pull request by khanzadyan**
Details of the product update v22.44 from 3rd of November 2022.

### CODE REVIEW

**Review by HannaTrotsenko — APPROVED** [review]

File: content/_releases/22/44.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+releaseDate: 2022-11-03
+redirect_from:
+  - /releases/22.44.html
+---
+
+## Improvements
+
+### Component repositories: New design & improved functionality
+
+We continue to update our User Interface to improve ease of use. This time it is
```
Comment (KMobbs) [inline content/_releases/22/44.md:17]: In this release we update the UI of the Component repository pages.. When you navigate to Developer Teams on the side menu and then select a Component repository you will notice the new design.

File: content/_releases/22/44.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+the UI of every component repository that is updated. When you navigate to
+Development > Developer Teams > Random component your team you will notice new
+uniform design:
+
+{% include img.html max-width="80%" url="/assets/img/RN/2244/RN2244-repository-view.png" title="Component repositories: New design Inside" %}
+
+From each component repository page you can navigate to the environment variables
+page where you can add new, change or delete the existing environment variables
```
Comment (KMobbs) [inline content/_releases/22/44.md:25]: new, **change,** or delete

File: content/_releases/22/44.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements included in this release.
+
+### RabbitMQ HAProxy in kubernetes cluster
+
+The HAProxy for RabbitMQ cluster will now run as a separate service in the platform
```
Comment (KMobbs) [inline content/_releases/22/44.md:41]: ...instead of as a single virtual machine.

File: content/_releases/22/44.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+{% endif %}
+
+
+## Fixed bugs
+
+*   Fixed the user interface bug when flow status changes were not reflected on the flows page.
+*   Addressed the problem when removing the user from the workspace or contract would take more than 4 minutes. Now the process takes less than a second.
+*   Fixed the bug when API would return `5xx` error when you use a space in the repository name while creating a component repository. Now the UI will disable the save button and show en error message `Repository name contains invalid characters. Letters, digits, - and _ are allowed only`
+*   Addressed the bug when identity keys page (SSH Keys) would not load in case when a key without title were saved previously. Now the title is a required field:
```
Comment (KMobbs) [inline content/_releases/22/44.md:70]: ...would not load when a key was previously saved without a title. The title is now a required field.

File: content/_releases/22/44.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+section: Product Updates
+category: product-updates
+releaseDate: 2022-11-03
+redirect_from:
+  - /releases/22.44.html
+---
+
+## Features
+
+### SaaS Marketplace Enablement: Introduction
```
Comment (KMobbs) [inline content/_releases/22/44.md:15]: COMING SOON - Improved Recipes and Embedding

File: content/_releases/22/44.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+In this release we make available a number of new API features that will enable
```
Comment (KMobbs) [inline content/_releases/22/44.md:17]: will soon enable

**Pull request merged**